src/js/index.js

```diff
@@ -79,3 +79,23 @@ export function getElementsBondsData(element1, element2, order = undefined) {
     const bondsData = filterBondsDataByElementsAndOrder(ELEMENT_BONDS, element1, element2, order);
     return bondsData.length ? bondsData : [defaultElementsBondsData];
 }
+
+/**
+ * Returns the atomic radius for an element based on it's atomic symbol (i.e. 'O' for Oxygen)
+ * The atomicRadius returned from PERIODIC_TABLE is in units of picometers (pm).
+ * The radius is returned in units of Angstroms. To convert pm to Angstroms, multiply the pm value by 0.01.
+ *
+ * If the element does not exist inside the PERIODIC_TABLE then the atomicRadius is returned as 1.0.
+ * A value of 1.0 is chosen so that if this value is used to set a new lattice size, then the lattice size
+ * will not collapse to 0, but will instead have a size of at least 1.0 angstrom.
+ *
+ * @param elementSymbol {String}
+ * @returns {Number}
+ */
+export function getElementAtomicRadius(elementSymbol) {
+    const elementName = Object.keys(PERIODIC_TABLE).find(key => elementSymbol === PERIODIC_TABLE[key].symbol);
```

Comment:
I think you can just use `ELEMENTS_BY_SYMBOL` to look up the atomic radius without searching through `PERIODIC_TABLE`.